libpoolprotocols/PoolManager.h

```diff
@@ -31,6 +31,7 @@ struct PoolSettings
     std::string hashRateId =
         h256::random().hex(HexPrefix::Add);  // Unique identifier for HashRate submission
     unsigned connectionMaxRetries = 3;  // Max number of connection retries
+    unsigned delayBeforeRetry = 10;     // Delay seconds before connect retry
```

Comment:
Default should be lower. 0-5 seconds tops

Reply:
got it

Reply:
5 seconds ok?

Reply:
I think @AndreaLanfranchi prefers 0 seconds, like it is currently. (https://github.com/ethereum-mining/ethminer/pull/1813#issuecomment-452638852)

Reply:
I am afraid of the retry mechanism wont works by default if we set it to 0